### PR TITLE
Feat/simplify search n filter

### PR DESCRIPTION
**Pull request by ben-jamming-reilly**
_Resolves #114_

### CODE REVIEW

File: src/gigs-board/components/layout/SearchByAuthor.jsx [hunk 1]
```diff
@@ -59,15 +59,54 @@ const onChangeAuthor = (selectedAuthors) => {
...
+  // );
+
+  return (
+    <div
+      // style={{ width: "10rem", maxHeight: "10rem", overflowY: "scroll" }}
+      // id="basic-typeahead-single"
+      aria-label="menu-options"
+      class="rbt-menu dropdown-menu show"
+      role="listbox"
+      // style="position: absolute; inset: 0px auto auto 0px; display: block; max-height: 300px; overflow: auto; transform: translate3d(0px, 38px, 0px); width: 160px;"
```
Comment (frol) [inline src/gigs-board/components/layout/SearchByAuthor.jsx:74]: I feel that this is still a draft PR given that there are some commented-out parts in the code.

File: src/gigs-board/components/layout/Controls.jsx [hunk 2]
```diff
@@ -45,141 +45,17 @@ function href(widgetName, linkProps) {
...
-      </div>
-    </div>
+  <div class="d-flex flex-row-reverse">
+    <a
+      class="btn btn-light"
+      style={{
+        backgroundColor: "#0C7283",
+        color: "#f3f3f3",
+      }}
+      href="gigs-board.pages.Create"
```
Comment (frol) [inline src/gigs-board/components/layout/Controls.jsx:55]: Use `href` helper (see https://github.com/near/neardevhub-widgets/pull/122#discussion_r1198920006 for more details)

File: src/gigs-board/components/layout/Controls.jsx [hunk 3]
```diff
@@ -45,141 +45,17 @@ function href(widgetName, linkProps) {
...
+    >
+      <i class="bi bi-plus-circle-fill"></i>
+      Submit Idea / Solution
```
Comment (frol) [inline src/gigs-board/components/layout/Controls.jsx:58]: Can we make the search inputs and this button to be on the same line?

image

**Review by frol — CHANGES_REQUESTED** [review]

File: src/gigs-board/pages/Create.jsx [hunk 4]
```diff
@@ -0,0 +1,526 @@
+/* INCLUDE: "common.jsx" */
```
Comment (frol) [inline src/gigs-board/pages/Create.jsx:1]: Just so everyone is on the same page, it seems we cannot merge this PR without #122.

**Comment by ben-jamming-reilly** [comment]
My benreilly.near widgets were out of date. Here is the current version
[Video](https://www.loom.com/share/6d5a93c096f348b89c7a90f23f17e808)

Below is the update widgets with both the new search and create post experience
[Test](https://near.org/benreilly.near/widget/gigs-board.pages.Feed?nearDevGovGigsContractAccountId=devgovgigs.near&nearDevGovGigsWidgetsAccountId=benreilly.near)

**Comment by frol** [comment]
It seems that all the changes were done in #122 and this PR only had an incomplete implementation of `customMenu` on the `SearchByAuthor` component, so I am closing this PR.

**Pull request closed without merging**